packages/external-db-config/lib/factory.js

```diff
@@ -58,6 +64,9 @@ const create = () => {
         case 'airtable':
           internalConfigReader = new gcp.GcpAirtableConfigReader()
           break;
+        case 'dynamo':
+          internalConfigReader = new aws.AwsDynamoConfigReader(region) 
+          break
```

Comment:
gcp doesn't have dynamo

Reply:
it's not in gcp, it's in azure for the e2e test. 
` process.env.CLOUD_VENDOR = 'azr'`   (all e2e tests are on azure)